apps/tables/src/components/TablesTable.vue

```diff
@@ -8,7 +8,9 @@
     </thead>
     <tr v-for="table in tables" :key="table.id">
       <td>
-        <router-link :to="table.id">{{ table.label }}</router-link>
+        <router-link :to="table.id">{{
+          table.label === null || table.id
```

Comment:
Don't you mean "table.label || table.id" ? this will display "true" if the label is null, which doesn't seem correct.